api/apps/api/src/modules/projects/blm/set-project-blm-handler.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { SetProjectBlmHandler } from '@marxan-api/modules/projects/blm/set-project-blm-handler';
+import { ProjectBlmRepo } from '@marxan-api/modules/blm';
+import { SetProjectBlm } from '@marxan-api/modules/projects/blm/set-project-blm';
+import { right } from 'fp-ts/Either';
+import { Test } from '@nestjs/testing';
+import { MemoryProjectBlmRepository } from '@marxan-api/modules/blm/values/repositories/memory-project-blm-repository';
+
+// TODO: Should we remove this unitary test and make a proper e2e test?
+describe.skip('set-project-blm-handler', () => {
```

Comment:
We would extract calculations to standalone dependency and keep unit tests for it, while e2e/integration for whole module.

Reply:
@AaronPerezPerez I agree with Kamil's suggestion here (for a later PR), however, this test looks useful already, in the meanwhile - can we un-skip it (and maybe give it a clearer description)?

Reply:
In the following PRs, I refactorized this and added a test to the domain service in charge of the calculations, and also an e2e test for the related endpoint. Should we still keep this?